internal/functions/registernotification/register-notification.go

```diff
@@ -137,3 +148,17 @@ func handleForFUID(ctx context.Context, fuid string) error {
 
 	return nil
 }
+
+func parseExpirationTime(idToken string) int64 {
+	token, _ := jwt.Parse(idToken, func(token *jwt.Token) (interface{}, error) {
+		return []byte("hello-world"), nil
+	})
+
+	// We certainly got validation error but we don't care. It's just about whether we are able to parse the token or not.
+
+	if token == nil || token.Claims == nil {
+		return 0
+	}
+
+	return int64(token.Claims.(jwt.MapClaims)["exp"].(float64))
```

Comment:
is it safe to access `token.Claims.(jwt.MapClaims)["exp"]` without checking for nil?

Reply:
Good point... Even though I strongly believe it _will_ be there, it's not safe.

Reply:
I'll use the technique shown in [jwt-go](https://github.com/dgrijalva/jwt-go/blob/master/map_claims.go#L22).